src/main/java/io/github/mzmine/gui/HeadLessDesktop.java

```
@@ -103,4 +104,8 @@ public void displayException(Exception e) {
     return null;
   }
 
+  public void openWebPage(String url) {
+//    Do nothing
```

You can `throw new UnsupportedOperationException();` instead